docs/java/how-to-spawn-a-workflow-execution-in-java.md

```diff
@@ -41,19 +91,37 @@ WorkflowStub workflowStub = client.newUntypedWorkflowStub(workflowType, workflow
 String result = workflowStub.getResult(String.class);
 ```
 
-### Synchronous start
+### Recurring start
 
-A Synchronous start initiates a Workflow and then waits for its completion. The started Workflow will not rely on the invocation process and will continue executing even if the waiting process crashes or was stopped.
+You can start a Workflow Execution on a regular schedule with [the CronSchedule option](distributed-cron).
 
-<!--SNIPSTART hello-world-project-template-java-workflow-initiator-->
-<!--SNIPEND-->
+### Calling other Workflows
 
-### Recurring start
+Use `ExternalWorkflowStub` to send Signals to other Workflow Executions, or cancel other running Workflow Executions from within a Workflow Definition.
+Workflows can also invoke, and send Signals to, other Workflows purely by name. This helps particularly for executing Workflows written in other language SDKs, as shown in the following example.
 
-You can start a Workflow Execution on a regular schedule with [the CronSchedule option](distributed-cron).
+```java
+@Override
+    public String exec() {
+
+        ExternalWorkflowStub externalGoWorkflowStub = Workflow.newUntypedExternalWorkflowStub("simple-workflow-go");
+
+        // Send 10 signals to Go Workflow
+        for(int i=0; i < 10; i++) {
+            externalGoWorkflowStub.signal("simplesignal", "Hello from Java Workflow: " + i);
+        }
+
+        // Receive 10 signals from Go Workflow
+        Workflow.await(() -> messageQueue.size() == 10);
+
+        String result = "";
+        for(String m : messageQueue) {
+            result += m + "\n";
+        }
+```
 
-### External Workflows
+Note that you can only send a Signal to other running Workflow Executions.
+To send Signals to a Workflow Execution whose status is unknown, use `SignalWithStart`. This method ensures that if the Workflow Execution is in a closed state, a new Workflow Execution is spawned and the Signal is delivered to the running Workflow Execution.
```

Comment:
Iirc you cannot do signalWithStart from within workflow code itself. You would need to do it from an activity (or client code). from within workflow code you would need to connect to running exec via external stub and then signal